code/style.css

```diff
@@ -1,4 +1,252 @@
-html {
-  background: #0025ff;
+body {
+  background: #fff;
+  color: #000;
+  margin: 0;
+}
+/* MOBILE FIRST */
+/* TOP NAVIGATION MOBILE */
+
+.header {
+  display: flex;
+  margin: 40px;
+}
+/*fixed navbar, seen when scrolling*/
+.nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  margin: auto;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.9);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  margin: 10px 40px;
+}
+
+/*does not show navigation item except logo i mobile view*/
+.navigation-container {
+  display: none;
+}
+
+/* BIG NEWS MOBILE */
+
+.big-news-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 10px;
+  padding: 0px 40px;
+}
+
+.big-news-text p {
+  margin: 0 0 10px 0;
+  color: lightgray;
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.big-news-img .img {
+  width: 100%;
+  height: 400px;
+}
+
+.img {
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 6px 12px -2px, rgba(0, 0, 0, 0.3) 0px 3px 7px -3px;
+  object-fit: cover;
+}
+
+.big-news-text h1 {
+  margin: 20px 0px;
+  font-size: 28px;
+}
+
+.button {
+  background-color: #1976D2;
   color: #fff;
+  border: none;
+  border-radius: 25px;
+  padding: 15px 25px;
+  text-align: center;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: #0D47A1;
+}
+
+/* SMAL NEWS MOBILE */
+
+.smal-news-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: 50px;
+  padding: 0px 40px;
+}
+
+.smal-news-heading h1 {
+  font-size: 28px;
+  margin: 0px;
+}
+
+.news-card-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.news-card {
+  display: flex;
+  flex-direction: column;
+  margin: 17px 0px;
+}
+
+.news-card .img {
+  width: 100%;
+  height: 300px;
+}
+
+.news-card h2 {
+  font-size: 18px;
+}
+
+.news-card p {
+  margin-top: auto;
+  color: lightgray;
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+/* FOOTER */
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 250px;
+  margin: 0px;
+}
+
+.credit-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
The footer could possibly use some twitching. Possibly a decreased height, but more importantly: since the footer only have one descendant, the `flex-direction: column;` won't do anything. Get rid of the `div` or move the `flex-direction: column;` to the `div` :)

<img width="397" alt="Skärmavbild 2021-02-03 kl  14 39 32" src="https://user-images.githubusercontent.com/51529201/106754758-a273ed80-662d-11eb-865e-4b3d8df909ad.png">